Certain-Chickens/CLI-Bangazon-Certain-Chickens
# Greg turner

**Description of Updates:**

- Corrected Orderz to have DateCreated instead of DateCompleted. 
- Changed DateCreated to handle DateTime instead of only text. 
- Updated the OrderzManager methods to work with database tables and entries. 
- OrderzManagerShould tests delete all data in the Orderz table prior to each test for a more controlled testing environment. 
- Changed the ListOrderz method to parallel ticket requirement of only pulling a list Orderz with a specified CustomerId which have a PaymentTypeId of other than null. 

**Note:** Still need to capture within the list a list of products and their details associated with each order. 

**Steps to test:**

1. git fetch --all
2. git checkout Greg-Turner
3. git pull origin Greg-Turner
4. Navigate to /CLI-Bangazon-Certain-Chickens/test/Bangazon.Tests
5. dotnet test
6. Confirm that 12 tests succeeded
